example/src/components/Footer.tsx

```diff
@@ -1,20 +1,26 @@
 import React from 'react';
-import { Alert, Keyboard } from 'react-native';
+import { Alert, Keyboard, Platform, StyleSheet, View } from 'react-native';
 import { Footer as StoryFooter } from 'react-native-story-view';
 import { Strings } from '../constants';
 import { FooterProps } from './types';
 
 const Footer = ({ userStories, story, progressIndex }: FooterProps) => (
-  <StoryFooter
-    onSendTextPress={() => {
-      Alert.alert(
-        `${Strings.messageSent} ${userStories?.username} id ${
-          story?.[progressIndex!].id
-        }`
-      );
-      Keyboard.dismiss();
-    }}
-  />
+  <View style={styles.container}>
+    <StoryFooter
+      onSendTextPress={() => {
+        Alert.alert(
+          `${Strings.messageSent} ${userStories?.username} id ${
+            story?.[progressIndex!].id
+          }`
+        );
+        Keyboard.dismiss();
+      }}
+    />
+  </View>
 );
 
+const styles = StyleSheet.create({
+  container: { bottom: Platform.OS === 'android' ? 50 : 0 }
```

Comment:
@sagar-simformsolutions Do we still need this platform-specific style?

Reply:
this can be user perspective, we can remove it if do not need it @mukesh-simform